asmcli/commands/validate.sh

```diff
@@ -55,6 +55,8 @@ validate_dependencies() {
         enable_service_mesh_feature
       fi
       if [[ "${CA}" == "managed_cas" ]]; then
+        info "Wait 3 minutes for GKE HUB API enablement to propagate to the systems ..."
+        sleep 180
```

Comment:
If there needs to be a sleep, if should go in one of the x_ functions specific to managed CAS. It makes me nervous putting it into a function that's called in the non-experimental install path.

Reply:
Done: it has gone to one of the x_ functions specific to managed CAS.